Rubberduck.Core/UnitTesting/TestEngine.cs

```diff
@@ -75,7 +74,7 @@ public void Run(IEnumerable<TestMethod> tests)
                     .Where(test => test.Declaration.QualifiedName.QualifiedModuleName.ProjectId == capturedModule.Key.ProjectId
                                 && test.Declaration.QualifiedName.QualifiedModuleName.ComponentName == capturedModule.Key.ComponentName);
 
-                var fakes = _fakesFactory.GetFakesProvider();
+                var fakes = _fakesFactory.GetFakes();
```

Comment:
Can I haz a cheese release please?

Reply:
This did also not expose a Release method before. the Fakes we inject do, though, so I'll add that :)